scripts/obtain/dotnet-install.sh

```diff
@@ -56,6 +56,59 @@ say_verbose() {
     fi
 }
 
+get_os_download_name_from_platform() {
+    eval $invocation
+
+    platform="$1"
+    case "$platform" in
+        "centos.7")
+            echo "centos"
+            return 0
+            ;;
+        "debian.8")
+            echo "debian"
+            return 0
+            ;;
+        "fedora.23")
+            echo "fedora.23"
+            return 0
+            ;;
+        "fedora.24")
```

Comment:
Currently this list serves two purposes: it does the mapping from `/etc/os-release` to the actual RID, but it also serves as a whitelist. If the RID Is not found in this list, the script errors out with `OS name could not be detected: $ID.VERSION_ID` message. It's easy enough to change this, but it's a change to existing functionality slightly tangential to --runtime-id flag. Should I open a separate PR for that or fix it here?